status/backend.json

```diff
@@ -82,6 +82,10 @@
     },
     "pn": {
       "enabled": true,
+      "min_app_version": {
+        "ios": "0.0.0.0",
```

Comment:
Should we set this to the current min app version? `2.35.0.1`

Reply:
Changed https://github.com/pagopa/io-services-metadata/pull/733/commits/dfdccfb6614fc815a9a704d68241be42bb5a1637